automatic/awscli/update.ps1

```diff
@@ -0,0 +1,31 @@
+import-module au
+
+$releases = 'https://github.com/aws/aws-cli/releases'
+
+function global:au_SearchReplace {
+   @{
+        ".\tools\chocolateyinstall.ps1" = @{
+            "(^[$]url\s*=\s*)('.*')"      = "`$1'$($Latest.URL32)'"
+            "(^[$]checksum\s*=\s*)('.*')" = "`$1'$($Latest.Checksum32)'"
+            "(^[$]url64\s*=\s*)('.*')"      = "`$1'$($Latest.URL64)'"
+            "(^[$]checksum64\s*=\s*)('.*')" = "`$1'$($Latest.Checksum64)'"
+        }
```

Comment:
Seems like you are missing the regexes that should replace the URLs specified in the `VERIFICATION.txt` file. You will need to include those so the URLs gets updated. Use other packages in this repository for examples of how that can be done.